test/extended/util/db/common.go

```diff
@@ -0,0 +1,53 @@
+package db
+
+import (
+	"os/exec"
+
+	"github.com/openshift/origin/test/extended/util"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+// PodConfig holds configuration for a pod
+type PodConfig struct {
+	Container string
+	Env       map[string]string
+}
+
+func getPodConfig(c kclient.PodInterface, podName string) (conf *PodConfig, err error) {
+	pod, err := c.Get(podName)
+	if err != nil {
+		return nil, err
+	}
+	env := make(map[string]string)
+	for _, container := range pod.Spec.Containers {
```

Comment:
@rhcarvalho Yes, I agree that it looks strange. @mnagy Could you comment it? Do we have a special reason for doing that?